src/components/Associator/UserPermissionsTable.tsx

```diff
@@ -10,11 +10,11 @@ const styles = {
   },
 };
 
-export default ({ associatedItems }) => (
+const UserPermissionsTable = ({ associatedItems }) => (
   <div style={{ flex: 1 }}>
     <Table singleLine>
       <Table.Body>
-        {associatedItems.map(item => (
+        {associatedItems.map((item) => (
```

Comment:
If you had to add these parentheses manually... there may be a prettier option to require this then it will get fixed whenever prettier runs (on file save). Maybe take a look and add it to the prettier config in the package.json

Reply:
No that's my prettier running...